developers/release.md

```diff
@@ -0,0 +1,348 @@
+
+```
+mod 'katello',   '>= 0.2.0 < 0.3.0'
+```
+
+### Step 6: Configure Koji
+
+Note that to perform this step, Foreman must have generated it's Koji build targets for the targeted Foreman release. This is due to how Katello tags inherit from some Foreman tags. Coordinate with the Foreman release nanny on when they plan to do that step and wait to do this step until then. Koji needs to have configuration added specific to the tags that the release will occur under. The first step is to add new build roots and tags using the following script:
+
+```
+https://github.com/Katello/katello-misc/blob/master/rel-eng/koji_create_buildroots.sh
+```
+
+We recommend running this script in logical chunks instead of attempting to run the entire thing at once.
+
+### Step 7: Setup Mash Scripts
+
+In order to generate the release repositories on Koji, a mash script for the version being released needs to be added. If you do not have direct access to the koji.katello.org box, you can request access or ask someone who does to perform this step for you.
+
+Copy previous mash script to new one
+
+```
+cp /usr/local/bin/katello-mash-split-2.0.py /usr/local/bin/katello-mash-split-2.1.py
+```
+
+Open new mash script and replace the version with the new one
+
+```
+vim /usr/local/bin/katello-mash-split-2.1.py
+:%s/2.0/2.1/g
+```
+
+Now do the same for all of the 2.0 mash files in `/etc/mash`, copying them to 2.1 files and updating the contents from 2.0 to 2.1. Finally, generate the first set of X.Y repositories on koji:
+
+```
+/usr/local/bin/katello-mash-split-2.1.py
+```
+
+The mash can be run via the release_mash_katello Jenkins job. However, the job first needs to be updated for the new version you are building by adding the Foreman and Katello versions to the configuration of the job through the Jenkins Web UI.
+
+### Step 8: Verify Repos
+
+At this point we want to verify the updates we made to the version branches by mashing and checking the repositories generated on koji.
+
+```
+/usr/local/bin/katello-mash-split-X.Y.py
+```
+
+View the generated repositories at `http://koji.katello.org/releases/yum/katello-X.Y` and ensure none of the packages have a git hash within the package name.
+
+### Step 9: Build Fresh X.Y Packages
+
+For each repository that was branched, and for each package within a given repository (e.g. katello has both the katello and rubygem-katello packages) we do a tito release koji. For example, the Katello repository:
+
+```
+cd katello/
+tito release koji
+cd deploy/
+tito release koji
+```
+
+NOTE: This step can only be done after you have created the X.Y tags.
+
+### Step 10: Update Repos RPM
+
+Now we update the repos file to point at where the released version of the repositories will exist for the X.Y release. 
+
+```
+cd katello-misc/repos
+export VER=X.Y
+sed -i "s#/nightly/#/$VER/#g" katello*.repo
```

Comment:
This doesn't update the name of the Katello repository, 

Maybe `sed -i "s#/[Nn]ightly/#/$VER/#g" katello*.repo` instead.